image.c

```diff
@@ -601,7 +600,7 @@ void img_render(img_t *img)
 			}
 			imlib_image_put_back_data(data);
 		} else {
-			c = win->win_bg;
+			XColor c = win->win_bg;
```

Comment:
Have we loosed our requirements to not have to declare all variables at the top of the method?

Reply:
Not entirely, but then there's this right above at line 588:

https://github.com/nsxiv/nsxiv/blob/2e2942d926087b1131ab6b453486d93cef97458e/image.c#L588-L592

I think we declaring tmp variables at the top of a block should be fine since there's existing code that does that and it's also is C89 compliant.